garage/plotter/plotter.py

```diff
@@ -21,13 +21,16 @@ class Op(Enum):
 
 Message = namedtuple("Message", ["op", "args", "kwargs"])
 
+__plotters__ = []
```

Comment:
This can be a class member of `Plotter` rather than package-global.

See https://stackoverflow.com/a/12102666 for a useful pattern.